### PR TITLE
add some fixes to the botor code

### PR DESCRIPTION
**Pull request by Th368MoJ**
Not sure if you want to see anymore botor stuff @mratford, so feel free to ignore this.

@ms130 this should fix the issues you outlined on slack. I can only see the two issues you initially pointed out, so if I've missed anything, please do say something.

### CODE REVIEW

File: source/documentation/appendix-docs/botor.md [hunk 1]
```diff
@@ -574,6 +582,10 @@ list_files_in_buckets <- function(bucket_filter = NULL, prefix = NULL,
   file_list <- dplyr::bind_rows(purrr::map(bucket_filter, 
                                            list_files_in_bucket))
   if(is.numeric(max)) file_list <- head(file_list, max)
+  # apply some finishing touches so it aligns with s3tools version
+  colnames(file_list) <- stringr::str_replace_all(colnames(file_list), historic_column_names)
+  file_list[["filename"]] = coalesce(stringr::str_extract(file_list$key, "[^\\/]+$"), stringr::str_replace_all(file_list$key, "\\/", ""))
```
Comment (mratford) [inline source/documentation/appendix-docs/botor.md:587]:
```suggestion
  file_list[["filename"]] = dplyr::coalesce(stringr::str_extract(file_list$key, "[^\\/]+$"), stringr::str_replace_all(file_list$key, "\\/", ""))
```

File: source/documentation/appendix-docs/botor.md [hunk 2]
```diff
@@ -649,14 +658,14 @@ s3tools::list_files_in_buckets('alpha-everyone') %>%
     dplyr::filter(grepl("iris",path)) # Use a regular expression
 # Using botor
 botor::s3_ls('s3://alpha-everyone') %>% 
-    dplyr::filter(grepl("iris",path))
+    dplyr::filter(grepl("iris",uri))
```
Comment (mratford) [inline source/documentation/appendix-docs/botor.md:662]: Good spot! Thanks.

**Review by mratford — CHANGES_REQUESTED** [review]
That's good, one small change requested.

**Review by ms130 — APPROVED** [review]
These changes look good to me

**Review by mratford — APPROVED** [review]

**Pull request merged**